### PR TITLE
adding polymon.yml to orchestration

### PR DESCRIPTION
**Pull request by telwell**

### CODE REVIEW

File: polymon.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      - "--insecure-transport"
+      - "--keyfile"
+      - "local_keyfile"
+      - "--poly-work"
+      - "local"
+      - "--dd-api-key"
+      - "${DD_API_KEY}"
+      - "--log-format"
+      - "json"
+      - "--testing"
```
Comment (ceseale) [inline polymon.yml]: Add new line here please

**Comment by ceseale** [comment]
poly_work: dev might be better than local in terms of naming because we tend to think of orchestration as `dev` and we might have a `dev` version of polyswarmd .ect in the future

File: polymon.yml (hunk 1 above)
Comment (telwell) [reply polymon.yml]: Fixed in latest pr.

**Comment by telwell** [comment]
agree with @ceseale's comment above, changed local->dev

**Comment by myhndl** [comment]
@telwell we don't want to remove `POLY_WORK`, we just don't want to invent a new environment for testing .We still need to pass in `POLY_WORK` else no one can test against anything other than staging.

File: polymon.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    image: "polyswarm/poly-mon"
+    depends_on:
+      - polyswarmd
+    environment:
+      - PASSWORD=password
+      - PYTHON_UNBUFFERED=1
+      - MALICIOUS_BOOTSTRAP_URL=https://corpora.sfo2.digitaloceanspaces.com/pdfs-1000
+      - ARCHIVE_PASSWORD=icanhazmalware
+      - POLY_SIDECHAIN_NAME=gamma
+      - POLY_WORK=stage
```
Comment (myhndl) [inline polymon.yml]: `POLY_WORK=${POLY_WORK}`

Comment (telwell) [reply polymon.yml]: fixed in commit, thanks

**Pull request merged**